scripts/gulpfiles/build_tasks.js

```diff
@@ -336,18 +336,47 @@ function buildDeps(done) {
     'tests/mocha'
   ];
 
-  const args = roots.map(root => `--root '${root}' `).join('');
-  execSync(
-      `closure-make-deps ${args} 2>/dev/null >'${DEPS_FILE}'`,
-      {stdio: 'inherit'});
+  function filterErrors(text) {
+    return text.split('\n')
+        .filter(
+            (line) => !/^WARNING /.test(line) ||
+                !(/Missing type declaration./.test(line) ||
+                  /illegal use of unknown JSDoc tag/.test(line)))
+        .join('\n');
+  }
 
-  // Use grep to filter out the entries that are already in deps.js.
-  const testArgs = testRoots.map(root => `--root '${root}' `).join('');
-  execSync(
-      `closure-make-deps ${testArgs} 2>/dev/null \
-           | grep 'tests/mocha' > '${TEST_DEPS_FILE}'`,
-      {stdio: 'inherit'});
-  done();
+  new Promise((resolve, reject) => {
+    const args = roots.map(root => `--root '${root}' `).join('');
+    exec(
+        `closure-make-deps ${args} >'${DEPS_FILE}'`,
+        {stdio: ['inherit', 'inherit', 'pipe']},
+        (error, stdout, stderr) => {
+          console.warn(filterErrors(stderr));
+          if (error) {
+            reject(error);
+          } else {
+            resolve();
+          }
+        });
+  }).then(() => new Promise((resolve, reject) => {
+    // Use grep to filter out the entries that are already in deps.js.
+    const testArgs =
+        testRoots.map(root => `--root '${root}' `).join('');
+    exec(
+        `closure-make-deps ${testArgs} 2>/dev/null\
+             | grep 'tests/mocha' > '${TEST_DEPS_FILE}'`,
+        {stdio: ['inherit', 'inherit', 'pipe']},
+        (error, stdout, stderr) => {
+          console.warn(filterErrors(stderr));
+          if (error) {
+            reject(error);
+          } else {
+            resolve();
+          }
+        });
+  })).then(() => {
+    done();
```

Comment:
Is this fixing the `prepare` error?

Reply:
Not as far as I know!

The switch to `Promise`s is to allow the non-`Sync` version of `exec` to be used (which in turn is so that we can capture `stderr` even when the child process does `exit(0)`).  We still need to make sure `done()` is called after all the rest of the work is done, though, as it was in the original version.  (We could of course alternatively return a vinyl stream, but that doesn't seem applicable for this task.)